site/_data/authorsData.json

```diff
@@ -177,8 +177,10 @@
     "twitter": "hochsays"
   },
   "jeffposnick": {
-    "homepage": "https://twitter.com/jeffposnick",
-    "twitter": "jeffposnick"
+    "homepage": "https://jeffy.info/",
+    "twitter": "jeffposnick",
+    "github": "jeffposnick",
+    "image": "image/FNkVSAX8UDTTQWQkKftSgGe9clO2/aNrmVViMdqMvdXZn0NHA.jpg"
```

Comment:
```suggestion
    "image": "image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/n1MbZiHKR9WnntmnPwcr.jpg"
```

Reply:
Per the [images and video](https://web.dev/handbook/markup-media/) guidelines, author avatars should be 384x384.

Reply:
I think our image CDN takes care of that, though? E.g., here's the HTML for the author image at https://developer.chrome.com/blog/insider-april-2022/

```html
<img alt="Ben Galbraith" class="flex-shrink-none height-600 width-600" decoding="async" height="40" loading="lazy" sizes="(min-width: 40px) 40px, calc(100vw - 48px)" src="https://wd.imgix.net/image/C8ys09lNbQOJLokQMhRtN2A0YU12/cySC1Gzh8NlFSsURpnpS.jpg?auto=format" srcset="https://wd.imgix.net/image/C8ys09lNbQOJLokQMhRtN2A0YU12/cySC1Gzh8NlFSsURpnpS.jpg?auto=format&amp;w=40 40w, https://wd.imgix.net/image/C8ys09lNbQOJLokQMhRtN2A0YU12/cySC1Gzh8NlFSsURpnpS.jpg?auto=format&amp;w=46 46w, https://wd.imgix.net/image/C8ys09lNbQOJLokQMhRtN2A0YU12/cySC1Gzh8NlFSsURpnpS.jpg?auto=format&amp;w=52 52w, https://wd.imgix.net/image/C8ys09lNbQOJLokQMhRtN2A0YU12/cySC1Gzh8NlFSsURpnpS.jpg?auto=format&amp;w=59 59w, https://wd.imgix.net/image/C8ys09lNbQOJLokQMhRtN2A0YU12/cySC1Gzh8NlFSsURpnpS.jpg?auto=format&amp;w=68 68w, https://wd.imgix.net/image/C8ys09lNbQOJLokQMhRtN2A0YU12/cySC1Gzh8NlFSsURpnpS.jpg?auto=format&amp;w=77 77w, https://wd.imgix.net/image/C8ys09lNbQOJLokQMhRtN2A0YU12/cySC1Gzh8NlFSsURpnpS.jpg?auto=format&amp;w=88 88w, https://wd.imgix.net/image/C8ys09lNbQOJLokQMhRtN2A0YU12/cySC1Gzh8NlFSsURpnpS.jpg?auto=format&amp;w=100 100w, https://wd.imgix.net/image/C8ys09lNbQOJLokQMhRtN2A0YU12/cySC1Gzh8NlFSsURpnpS.jpg?auto=format&amp;w=114 114w, https://wd.imgix.net/image/C8ys09lNbQOJLokQMhRtN2A0YU12/cySC1Gzh8NlFSsURpnpS.jpg?auto=format&amp;w=120 120w" width="40">
```